### DISCUSSION TITLE
change layout on daily notes

### DISCUSSION THREAD

**Discussion by fdeyber** [kudrykv/latex-yearly-planner#96] [General]
I removed the schedule successfully, now I would like to rearrange the layout to:
![image](https://github.com/kudrykv/latex-yearly-planner/assets/52584806/2a8efc90-0afe-4a8b-b3e2-2099c8ad844f)
 and maybe put the calendar on the top right corner or extend the todo lines. How can I do that?

**Comment by fdeyber** [comment]
I was able to get to this by messing with the tpl files and yaml
![image](https://github.com/kudrykv/latex-yearly-planner/assets/52584806/722360b3-aa96-458c-b3fb-421781a5eea1)

**Reply by enomis123** [reply]
Oh , I like this . Could you please share the cfg, tpl,… so I can try to re-generate as well . Thank you ,
